engine/src/core/graphics/rendering/rendering_device/renderer_2d.h

```diff
@@ -0,0 +1,39 @@
+//
+// Created by blinku on 07.08.2019.
+//
+
+#pragma once
+
+#include <memory>
+#include "batch_2d.h"
+
+namespace Bald::Graphics {
+
+    class Sprite;
+    class VertexArray;
+    class VertexBuffer;
+    class IndexBuffer;
+    class Shader;
+    class Camera2D;
+
+    struct Renderer2DData {
+        static inline std::unique_ptr<Batch2D> m_Batch = nullptr;
+        static inline const Camera2D* m_Camera = nullptr;
```

Comment:
why this is inline and static?

Reply:
To have the definition inside this little struct and not somewhere else, outside of the class's body.